ecommerce/Ecommerce.php

```diff
@@ -73,6 +74,14 @@ public function __construct( $register_hooks = true ) {
 		new HooksHandler();
 		new EmailController();
 		new PaymentHandler();
+
+		// Globals.
+		add_action(
+			'init',
+			function() {
+				$GLOBALS['coupon_apply_error_msg'] = ( new CouponModel() )->get_coupon_failed_error_msg( 'not_applicable' );
```

Comment:
Use `tutor_` prefix to this global variable to prevent conflict

Reply:
Added